jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -373,10 +373,6 @@ var module = module;
           location: extraInfo['bzr-url']
         }
       };
-      if (meta['charm-related']) {
-        this._lowerCaseKeys(meta['charm-related'], meta['charm-related']);
-        processed.relatedCharms = meta['charm-related'];
```

Comment:
Nope, just completely missed that as I was grepping for 'charm-related'. Thanks!